app/src/main/java/org/sagebionetworks/research/mpower/studyburst/StudyBurstItem.kt

```diff
@@ -0,0 +1,11 @@
+package org.sagebionetworks.research.mpower.studyburst
+
+class StudyBurstItem {
```

Comment:
Any time you have a class that is made up of primarily vars and vals, you mine as well define it like this...

data class StudyBurstItem {
   // vars and vals
}

[From https://kotlinlang.org/docs/reference/data-classes.html#data-classes]

We frequently create classes whose main purpose is to hold data. In such a class some standard functionality and utility functions are often mechanically derivable from the data.

You get all these additional function calls available
...

equals()/hashCode() pair;
toString() of the form "User(name=John, age=42)";
componentN() functions corresponding to the properties in their order of declaration;
copy() function (see below).

Reply:
This class should also extend the TaskInfo class